src/main/resources/data/galacticraft/recipes/crafting_table/decoration/dark/dark_decoration.json

```diff
@@ -0,0 +1,20 @@
+{
+  "type": "minecraft:crafting_shaped",
+  "pattern": [
+    "SS ",
+    "SS ",

```

Comment:
I believe if you remove the spaces at the end, you will be able to put the materials on the left or right side of the crafting table.